src/html.rs

```diff
@@ -478,18 +478,28 @@ impl<'o, 'c: 'o> HtmlFormatter<'o, 'c> {
             NodeValue::List(ref nl) => {
                 if entering {
                     self.cr()?;
-                    if nl.list_type == ListType::Bullet {
-                        self.output.write_all(b"<ul")?;
-                        self.render_sourcepos(node)?;
-                        self.output.write_all(b">\n")?;
-                    } else if nl.start == 1 {
-                        self.output.write_all(b"<ol")?;
-                        self.render_sourcepos(node)?;
-                        self.output.write_all(b">\n")?;
-                    } else {
-                        self.output.write_all(b"<ol")?;
-                        self.render_sourcepos(node)?;
-                        writeln!(self.output, " start=\"{}\">", nl.start)?;
+
+                    match nl.list_type {
+                        ListType::Bullet => {
+                            self.output.write_all(b"<ul")?;
+                            if nl.is_task_list {
+                                self.output.write_all(b" class=\"contains-task-list\"")?;
```

Comment:
I'm going to advocate again for `task-list`. This doesn't "contain" a task list, it "is" a task list. I don't think we need to follow GH on this. https://github.com/jgm/commonmark-hs also uses `task-list`.